content/cookbook/handling-forms.md

```diff
@@ -0,0 +1,178 @@
+
+{{<highlight html>}}
+<textarea name="textarea-input" ></textarea>
+{{</highlight>}}
+
+{{<highlight javascript>}}
+await _setValue(_textarea('textarea-input'), 'Some longer text maybe...')
+{{</highlight>}}
+
+### Textboxes
+Similar to `<textarea>` elements, text input fields can easily be filled with text by combining the [_setValue() action](https://sakuli.io/apidoc/sakuli-legacy/interfaces/actionapi.html#_setvalue) with the [_textbox() accessor](https://sakuli.io/apidoc/sakuli-legacy/interfaces/accessorapi.html#_textbox). The `_textbox()` accessor is configured to catch `<input>` fields without any `type` or with `type=text`. 
+
+{{<highlight html>}}
+<input type="text" name="first-name" />
+<input name="last-name" />
+<input id="Street" />
+{{</highlight>}}
+
+{{<highlight javascript>}}
+await _setValue(_textbox('first-name') ,"Bill")
+await _setValue(_textbox('last-name') ,"Jobs")
+await _setValue(_textbox('street') ,"Money Blvd.")
+{{</highlight>}}
+
+
+### Emails
+Similar to `<input type="text">` elements, E-Mail input fields can easily be filled with an E-Mail address by combining the [_setValue() action](https://sakuli.io/apidoc/sakuli-legacy/interfaces/actionapi.html#_setvalue) with the [_emailbox() accessor](https://sakuli.io/apidoc/sakuli-legacy/interfaces/accessorapi.html#_emailbox). The `_emailbox()` accessor is configured to catch `<input>` fields or with `type=email`. 
+
+{{<highlight html>}}
+<input type="email" name="mail-input" />
+{{</highlight>}}
+
+{{<highlight javascript>}}
+await _setValue(_emailbox('mail-input'), 'do-not-spam@sakuli.io')
+{{</highlight>}}
+
+
+### Passwords
+Password fields are a little different compared to standard input fields as they are hiding the provided input. Sakuli provides a [_password() accessor](https://sakuli.io/apidoc/sakuli-legacy/interfaces/accessorapi.html#_password) to access `<input>` fields with `type=password`. In combination with the [_setValue() action](https://sakuli.io/apidoc/sakuli-legacy/interfaces/actionapi.html#_setvalue), it is easily possible to set passwords in those fields.
+
+{{<highlight html>}}
+<input type="password" name="password-input" />
+{{</highlight>}}
+
+{{<highlight javascript>}}
+await _setValue(_password('password-input'), "$ecret")
+{{</highlight>}}
+
+## Buttons
+### Standard buttons
+Just as for other elements, Sakuli provides a [_button accessor](https://sakuli.io/apidoc/sakuli-legacy/interfaces/accessorapi.html#_button). The difference to e.g. input fields is the action you perform on button elements. Instead of setting a text value you would rather click the button. To do so, Sakuli provides a [_click action](https://sakuli.io/apidoc/sakuli-legacy/interfaces/actionapi.html#_click).
```

Comment:
```suggestion
Just like other elements, Sakuli provides a [_button accessor](https://sakuli.io/apidoc/sakuli-legacy/interfaces/accessorapi.html#_button). The difference to e.g. input fields is the action you perform on button elements. Instead of setting a text value, you would rather click the button. To do so, Sakuli provides a [_click action](https://sakuli.io/apidoc/sakuli-legacy/interfaces/actionapi.html#_click).
```